src/libAtomVM/term.c

```diff
@@ -205,7 +205,7 @@ int term_funprint(PrinterFun *fun, term t, const GlobalContext *global)
     } else if (term_is_local_port(t)) {
         // Update also PORT_AS_CSTRING_LEN when changing this format string
```

Comment:
Just for reference to save reviewers the trouble of looking this up, these changes match the length that was already defined for `PORT_AS_CSTRING_LEN` (lines 161-168 of term.h):

```
// Local port is at most 19 bytes:
// 2^28-1 = 268435455 (9 chars)
// "#Port<0." ">\0" (10 chars)
// External port is at most 37 bytes:
// 2^26-1 = 67108863 (8 chars) (node, atom index)
// 2^64-1 = 18446744073709551615 (20 chars)
// "#Port<" "." ">\0" (9 chars)
#define PORT_AS_CSTRING_LEN 37
```